code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -57,6 +57,10 @@
 	message = "mumbles"
 	emote_type = EMOTE_AUDIBLE
 
+/datum/emote/living/carbon/human/givesItem
+	key = "givesitem"
+	message = "offers you the item"
```

Comment:
```suggestion
	message = "offers an item"
```
This will appear to everyone so it shouldn't say "you".
Also the key should be "offer" I think rather than "givesitem"

Reply:
That makes sense!